src/orderly/cli/__init__.py

```diff
@@ -0,0 +1,204 @@
+import sys
+from collections import Counter
+
+import click
+
+import outpack.search
+from orderly.cli.options import (
+    NumericalParamType,
+    PropagatingGroup,
+    with_root,
+    with_search_options,
+)
+from orderly.run import orderly_run
+from outpack.init import outpack_init
+from outpack.location import (
+    outpack_location_add,
+    outpack_location_list,
+    outpack_location_remove,
+    outpack_location_rename,
+)
+from outpack.util import format_list
+
+
+@click.group(cls=PropagatingGroup)
+@with_root(propagate=True)
+def cli():
```

Comment:
Can you override the default help slightly?

```
(outpack) rfitzjoh@wpia-dide300:~/Documents/src/outpack-py$ orderly --help
Usage: orderly [OPTIONS] COMMAND [ARGS]...

Options:
  --help  Show this message and exit.

Commands:
  init      Initialize at new orderly repository in PATH.
  location  Manage remote locations.
  run       Run a report.
  search    Search existing packets by query.
```

it would be good to get across that `orderly init --help` would provide more help